src/frontend.cpp

```diff
@@ -648,10 +648,10 @@ void create_message_box(const char *title, const char *line1, const char *line2,
     turn_on_menu(GMnu_MSG_BOX);
 }
 
-short game_is_busy_doing_gui(void)
+short game_is_busy_doing_gui_for_player(PlayerNumber plyr_idx)
```

Comment:
This change comes from a branch you merged in before it was merged into master. And it was reverted there before it was merged because it did not work. So it needs to be reverted here too. There's several places where this happened.